src/Tizen.Network.Stc/Tizen.Network.Stc/StcEnumerations.cs

```diff
@@ -0,0 +1,258 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Tizen.Internals.Errors;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// Enumeration for Stc time period.
+    /// </summary>
+    public enum StcTimePeriod
+    {
+        /// <summary>
+        /// Unknown
+        /// </summary>
+        Unknown = 0,
+        /// <summary>
+        /// Hour
+        /// </summary>
+        Hour = 3600,
+        /// <summary>
+        /// Day
+        /// </summary>
+        Day = 86400,
+        /// <summary>
+        /// Week
+        /// </summary>
+        Week = 604800,
+        /// <summary>
+        /// Month
+        /// </summary>
+        Month = 2419200
+    }
+
+    /// <summary>
+    /// Enumeration for network interface type.
+    /// </summary>
+    public enum StcInterfaceType
+    {
+        /// <summary>
+        /// Network interface type is Unknown.
+        /// </summary>
+        Unknown = 0,
+        /// <summary>
+        /// Network interface type is datacall. 
+        /// </summary>
+        Datacall,
+        /// <summary>
+        /// Network interface type is  wifi.
+        /// </summary>
+        Wifi,
+        /// <summary>
+        /// Network interface type is  wired.
+        /// </summary>
+        Wired,
+        /// <summary>
+        /// Network interface type is bluetooth.
+        /// </summary>
+        Bluetooth,
+        /// <summary>
+        /// Network interface type is all.
+        /// </summary>
+        All
+    }
+
+    /// <summary>
+    /// Enumeration for network roaming type
+    /// </summary>
+    public enum StcRoamingType
+    {
+        /// <summary>
+        /// Not in roaming.
+        /// </summary>
+        Disabled,
+        /// <summary>
+        /// In roaming.
+        /// </summary>
+        Enabled
+    }
+
+    /// <summary>
+    /// Enumeration for hardware network protocol type. 
+    /// </summary>
+    public enum StcProtocolType
```

Comment:
How about rename to `NetworkProtocal`

1. Stc - no need, because already including `Stc` in namspace
2. Type - no need it, Everything is type
3. Network - need it becuase it is a category of network protocol

Reply:
Applied it for every enum.